src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1502,16 +1508,23 @@ export default {
 				this.basketBalancing = false;
 			}
 		},
-		checkoutLightboxClosed() {
+		async checkoutLightboxClosed() {
 			this.checkoutVisible = false;
+			// gaurd against navigation reset if hash is already empty
+			if (this.$router?.hash !== '') {
+				await this.$router.push(this.adjustRouteHash('')).catch(() => {});
+			}
 			this.handleScrollPosition();
-			this.$router.push(this.adjustRouteHash('')).catch(() => {});
```

Comment:
Once the event emitter was in place, this turned out to be an additional breaking issue resetting the page position after the scroll 🤦